src/controller/ticket.js

```diff
@@ -10,7 +10,7 @@ const bookticket=async(req,res)=>{
         if(availability.avaiableSeat.length>0 && ticketdetails.seatcount<=availability.avaiableSeat.length){
```

Comment:
Typo in avaiableSeat. And camelCase needed.

Reply:
typo and camelCase are changed